src/js/common/components/RequiredLoginView.jsx

```diff
@@ -103,7 +103,7 @@ RequiredLoginView.propTypes = {
   authRequired: PropTypes.number.isRequired,
   serviceRequired: PropTypes.string.isRequired,
   userProfile: PropTypes.object.isRequired,
-  loginUrl: PropTypes.string,
+  loginUrl: PropTypes.object.isRequired
```

Comment:
`RequiredLoginView` shouldn't require a `loginUrl` because of the times when the user is already logged in at the required LOA.  In these cases, the `RequiredLoginView` will simply render the child components - no `loginUrl` necessary.